package.json

```diff
@@ -38,7 +38,9 @@
     "eslint-config-airbnb": "^17.1.0",
     "eslint-config-airbnb-base": "^13.1.0",
     "eslint-plugin-import": "^2.17.2",
-    "nodemon": "^1.18.11"
+    "nodemon": "^1.18.11",
+    "sinon": "^7.3.2",
+    "sinon-chai": "^3.3.0"
```

Comment:
@Jonathan4github , Please remove all caret (ˆ) symbols in the dependency and the dev dependencies.. this will ensure the module versions are locked